TodoList/TodoList.Api.Tests/Utils/ItemCompare.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Tests.Utils
+{
+    public class ItemCompare : IEqualityComparer<Item>
```

Comment:
Since it extends `IEqualityComparer`, I would expect the class to be suffixed `Comparer` as well. Besides, `ItemCompare` is not a noun or combination of an adjective and a noun but a noun and a verb - this is a connection more fitting for a method (were the order opposite (`CompareItem`)).